submission/pipelines/generate_pipelines.py

```diff
@@ -543,7 +546,8 @@ def main():
         write_pipeline_for_submission(os.path.join(byu_dir, __metafeature_path__), str(__metafeature_version__), pipeline_json_structure, problem_name)
 
     # add other best pipelines
-    add_best_pipelines(byu_dir)
+    # TODO: update the experimenter to produce valid pipelines
+    # add_best_pipelines(byu_dir)
```

Comment:
I know there is a reason for this, but I can't remember... Can you remind me?

Reply:
Yes, once we update the experimenter and can get valid pipelines.  Right now we have to manually update them after we grab them